Sammuel09/Fast-Food-Fast
# #160704316 Get all orders endpoint (Admin)

#### What does this PR do?
Allows the admin to get all orders

#### Description of Task to be completed?
- Admin can get all orders
- The route is authenticated such that only admin can access the route.

### How should this be manually tested?
First, sign in to the app as ADMIN by making a POST request to https://sammie-fast-food-fast.herokuapp.com/api/v1/auth/login

Use the following data:
`username: samuel`
`password: samuel`


Copy the token received in the response in the signup process and add it as part of the request headers in this format:
`x-access-token: token`

On Postman, visit https://sammie-fast-food-fast.herokuapp.com/api/v1/orders
Then make a GET Request 
Expected Data to be returned:
`status: string`,
`data: array`,
`message: string`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2196965/stories/160704316
